node/db/src/sled.rs

```diff
@@ -0,0 +1,85 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+#![cfg(feature = "sled")]
```

Comment:
Isn't this implied by the `#[cfg(feature = "sled")]` on `pub mod sled;`?

Reply:
ah, yeah, I had it this way before and forgot to remove (I'll remove this as well as I didn't remove from the rocksdb redundant one as well)